omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/PersonController2_2Test.java

```diff
@@ -75,6 +77,19 @@ public void shouldEditAPerson() throws Exception {
         assertTrue(person.isDead());
         assertNotNull(person.getCauseOfDeathNonCoded());
     }
+
+    @Test
+    public void shouldEditAPersonBirthTime() throws Exception {
+        Person person = service.getPersonByUuid(getUuid());
+        assertNull(person.getBirthtime());
+        String json = "{\"birthtime\": \"1971-02-08T20:20:00.000\"}";
+        SimpleObject response = deserialize(handle(newPostRequest(getURI() + "/" + getUuid(), json)));
+        assertNotNull(response);
+        Object responsePersonContents = PropertyUtils.getProperty(response, "person");
+        assertNotNull(person.getBirthtime());
+        assertNotNull(responsePersonContents);
+        assertEquals("1971-02-08 08:20:00", new SimpleDateFormat("yyyy-MM-dd hh:mm:ss").format(person.getBirthtime()));
```

Comment:
At the beginning of this test, can you assert that the birthtime value was different from this?

Reply:
at the begining person.getBirthtime() gives null value. Initially birthtime of that person is not set and after that birthtime got added to the json value.

Reply:
so if i add ` assertNotEquals("1971-02-08 08:20:00", new SimpleDateFormat("yyyy-MM-dd hh:mm:ss").format(person.getBirthtime()));` this line at the beginning of test it is giving null pointer exception.

Reply:
Awesome! 👍

Reply:
@dkayiwa is this the released version of openmrs-module-webservices?

Reply:
This is the latest released version: https://addons.openmrs.org/show/org.openmrs.module.webservices-rest